themes/medius/scss/base.scss

```
@@ -0,0 +1,25 @@
+.base-container {
+  @media (min-width: $desktops) {
+    display: flex;
+    flex-wrap: wrap;
+    margin-top: 72px;
+    padding: 0 24px;
+  }
+
+  .sidebar-container {
+    display: none;
+
+    @media (min-width: $desktops) {
+      @include make-col(2);
+
+      display: block;
+    }
+  }
+
+  .page-container{
+    @media (min-width: $desktops) {
+      @include make-col(6);
+      @include make-col-offset(1);
+    }
+  }
+}
```

It does not need to be nested in `base-container`


```suggestion
}

.sidebar-container {
  display: none;

  @media (min-width: $desktops) {
    @include make-col(2);

    display: block;
  }
}

.page-container{
  @media (min-width: $desktops) {
    @include make-col(6);
    @include make-col-offset(1);
  }
}
```